src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2106,6 +2107,57 @@ bool AdaptDispatch::DeleteLine(const VTInt distance)
     return true;
 }
 
+// Routine Description:
+// - Internal logic for adding or removing columns in the active screen buffer.
+// Parameters:
+// - delta - Number of columns to modify (positive if inserting, negative if deleting).
+// Return Value:
+// - <none>
+void AdaptDispatch::_InsertDeleteColumnHelper(const VTInt delta)
```

Comment:
Actually now that I've had a look at them, all the `int32_t` parameters should really be `VTInt` - that's what they already are in the header! I think originally the VT parameters where coming in as an unsigned int of some sort, so we had to cast to the signed `int32_t` when they were used as a delta. But since `VTInt` is now signed, that's no longer necessary.